brackend/app.py

```diff
@@ -10,6 +10,7 @@
 # App Blueprints: Blueprint for each resource
 from brackend.endpoints.tournaments.tournaments import tournament_bp
 from brackend.endpoints.users import users_bp
+from brackend.endpoints.brackets.brackets import brackets_bp
```

Comment:
is it necessary for brackets/tournaments for that matter to be in its own folder and not just a file in the endpoints dir? That would clean up the imports a tad

Reply:
Yeah fair point - it's definitely not necessary. This is me leaning more towards many small files (but they'll grow!) - and trying to keep them co-located by their resource/concern. But actually I think you're right and it would make sense to instead have the repositories pulled out so we'd have:

brackend/endpoints/brackets.py
brackend/endpoints/tournaments.py
... etc for the api layer

then 
brackend/repositories/BracketsRepository.py
brackend/repositories/TournamentRepository.py

I've just been trying to keep all the resources a bit more silo'd, thinking that it'd make sense to have api/repo for the same resource located together, but doesn't seem as nice now

Reply:
Going to merge this as-is, but I've filed a trello card for doing some reorganization once everything in flight has been merged: https://trello.com/c/eI77Hw7a/39-repository-and-endpoint-re-org